src/pages/dev/amplifier/chain-integration/integrate-a-chain.mdx

```diff
@@ -11,6 +11,77 @@ The Axelar Virtual Machine (AVM) and Amplifier are currently under active develo
 - You should know how to deploy and interact with a CosmWasm contract. Refer to the [Osmosis CosmWasm testnet deployment tutorial](https://docs.osmosis.zone/cosmwasm/testnet/cosmwasm-deployment/) if you’d like a refresher — just replace `osmosisid` with `axelard` and use the `amd64` version if building on Linux and `arm64` version of the [contract optimizer](https://github.com/CosmWasm/optimizer) if you are working on a [Mac with Apple silicon](https://support.apple.com/en-us/116943). 
 - Run `--gas auto --gas-adjustment 1.5` to provide your transactions with sufficient gas.
 
+## Setup and configuration
```

Comment:
I don't think this belongs at the top of the file like this. `tofnd` is one of the last steps of the process (you have to fully integrate your chain first before you can start verifying transactions).